scripts/discord_webhook.py

```diff
@@ -0,0 +1,20 @@
+try:
+    import requests # Make sure requests is installed
+except Exception:
+    log = open("ERROR.txt", "w")
+    log.write("Requests is not installed, and is required to send discord webhook messages")
+    log.close()
+    exit()
+import sys
+
+# Arg 0 is the script name itself (discord_webhook.py)
+scriptname = sys.argv[0] # This exists so that the script name wont get sent in the final
+url = sys.argv[1] # First entry
+message = ""
+for item in sys.argv:

```

Comment:
This for loop can be replaced by `message = ' '.join(sys.argv[2:])`

Reply:
I will test this, if that works I will change it

Reply:
![image](https://user-images.githubusercontent.com/25063394/58197559-6ddac980-7cc4-11e9-9df5-c35cd7632167.png)